### DISCUSSION TITLE
Multiple Issues Regarding Transferring Ownership of Repository

### DISCUSSION THREAD

**Discussion by jpuangco-itrustcapital** [community/community#40585] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

I am being transferred ownership of a repository and on the collaborator's side it says- Your repository is being transferred to xxx. However, on my side I have not received any notifications.

https://docs.github.com/en/enterprise-server@3.3/repositories/creating-and-managing-repositories/transferring-a-repository

All I received was an invitation to collaborate. I tried to end up just cloning the repo but it asks me to log in and when I log in again, I run into a log in loop. Where it logs in and shows me a progress screen then sends me back to the login page again.

**Comment by jpuangco-itrustcapital** [comment]
Any help would be much appreciated as now the repository is in limbo and no one is able to modify it.

**Comment by ossanna16** [comment]
Hi there @jpuangco-itrustcapital and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
